src/Components/Categories.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] KIndly ensure you dont disable linters but rather endeavor to fix your linter errors because linters forces you to write clean code 👍🏽